src/scripts.js

```diff
@@ -27,10 +31,13 @@ window.addEventListener('load', function actOnLoad() {
   compareStepGoals();
   getDate();
   showHydrationData();
+  showSleepData();
 });
 
 dateInput.addEventListener('change', showHydrationData);
```

Comment:
We should definitely refactor these `dateInput` event listeners the same way we did for the `window`